src/script/squirrel_helper.hpp

```diff
@@ -299,8 +297,12 @@ namespace SQConvert {
 		/* Remove the userdata from the stack */
 		sq_pop(vm, 1);
 
-		/* Call the function, which its only param is always the VM */
-		return (SQInteger)(((Tcls *)real_instance)->*(*(Tmethod *)ptr))(vm);
+		try {
+			/* Call the function, which its only param is always the VM */
+			return (SQInteger)(((Tcls *)real_instance)->*(*(Tmethod *)ptr))(vm);
```

Comment:
What a horrid line of code, and with `static_cast`s it doesn't get any clearer.
```suggestion
			return static_cast<SQInteger>((static_cast<Tcls *>(real_instance)->*(*static_cast<Tmethod *>(ptr)))(vm));
```

This might be more legible:
```suggestion
			auto cls_instance = static_cast<Tcls *>(real_instance);
			auto method = *static_cast<Tmethod *>(ptr);
			return static_cast<SQInteger>((cls_instance->*method)(vm));
```